static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -175,7 +175,7 @@ define([
         var body;
 
         rules = rules || defaultRules;
-        body = rules.bodySelector ? document.querySelector(rules.bodySelector) : document;
+        body = (rules.bodySelector && document.querySelector(rules.bodySelector)) || document;
```

Comment:
hmm I see. Is the `.js-liveblog-body` class still applied to the body of the liveblog?